### ISSUE TITLE
ControllerSuite.TestExampleOutput intermittently fails with no output

### ISSUE THREAD

**Issue by titanous** [flynn/flynn#869]
```text
FAIL: test_controller.go:145: ControllerSuite.TestExampleOutput


test_controller.go:146:
    examples := s.generateControllerExamples(t)
test_controller.go:134:
    t.Assert(json.Unmarshal(stdout.Bytes(), &controllerExamples), c.IsNil)
... value *json.SyntaxError = &json.SyntaxError{msg:"unexpected end of JSON input", Offset:0} ("unexpected end of JSON input")
```

https://s3.amazonaws.com/flynn-ci-logs/20150129050420-0610e364-build-c5e5834890299309599fc5e1d129773f41ba1463-2015-01-29-05-09-55.txt

**Comment by titanous** [comment]
@jvatic Please ensure every error is checked, I'm not sure what plausible reasons are for this failing.

**Comment by titanous** [comment]
The log output was being discarded, fixed that in #904
